lib/bots/versionbot.ts

```diff
@@ -1266,6 +1270,28 @@ export class VersionBot extends ProcBot {
 
 				versionData.version = match[1];
 				versionData.files = moddedFiles;
+
+				// Now we have to add the PR number and URL to every log entry we've just added for
+				// the new version. This ensures that NotifyBot can apply release notes later on, if
+				// required.
+				const versions = contents.split('## ');
+
+				// Now find the right version entry.
+				for (let index = 0; index < versions.length; index += 1) {
```

Comment:
Given that you only use `index` to retrieve the elements of the array, this should probably be a `for(let version of versions)`